alert_group_nl_log2slack.py

```diff
@@ -383,18 +390,24 @@ def fetch_logs_and_publish_forever():
     already_published = set()
 
     while True:
+        date = datetime.datetime.now()
         data = set(fetch_logs_with_retry())
         not_published_yet = (data - already_published)
         print(f'data count: {len(data)}, new: {not_published_yet}')
         already_published = data
 
-        a_while_ago = (datetime.datetime.now() - datetime.timedelta(hours=4))
+        a_while_ago = (date - datetime.timedelta(hours=4))
         for record in sorted(not_published_yet, key=AlarmRecord.SORT_KEY):
             if record.datetime < a_while_ago:
                 print(f'skipping old: {record}')
             else:
                 send_slack_message(str(record))
                 print(f'sent message: {record}')
+        
+        with open(HEALTH_FILE, "w") as f:
+            f.seek(0)
+            f.write(str(date.isoformat(timespec="seconds")))
+            f.truncate()
```

Comment:
Can we just replace these four lines with `os.utime(HEALTH_FILE)` ?

Reply:
We definitely can